### DISCUSSION TITLE
【灌水】你们有没有过某绿色eh软件只能加载一张图的情况

### DISCUSSION THREAD

**Discussion by stormcat-efforia** [share121/inter-knot#3418] [其他]
每次都是只能加载一张图，还加载的贼慢，导致我看图的心情都没了

**Comment by Wax1234567** [comment]
你这个问题在绳网就可以解决<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >
（就在github里直接搜索eh，你应该会找到几个不同版本，你自己试试哪个比较好用喽）

**Comment by nninng** [comment]
同时开两个，哪边不动了就去另一边看。
